src/typing.py

```diff
@@ -1985,20 +1987,29 @@ class Employee(NamedTuple):
         The resulting class has one extra attribute: _field_types,
         giving a dict mapping field names to types.  (The field names
         are in the _fields attribute, which is part of the namedtuple
-        API.) Backward-compatible usage::
+        API.) Alternative equivalent keyword syntax is also accepted::
+
+            Employee = NamedTuple('Employee', name=str, id=int)
+
+        Backward-compatible usage::
```

Comment:
So "backward-compatible" sounds a bit pejorative. I think this should just be referred to as the way to do it for Python versions <= 3.5.